src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -710,15 +736,33 @@ public function release_bookmark( $name ) {
 		return true;
 	}
 
+	/**
+	 * Skips contents of generic rawtext elements.
+	 *
+	 * @since {VERSION NUMBER}
```

Comment:
```suggestion
	 * @since 6.4.0
```

Reply:
Unless you'd like it also in 6.3.x, as it qualifies as a bugfix? 🤔

Reply:
it's definitely a bug-fix, though it looks like I overlooked creating it that way in Trac. if we _can_ get it into 6.3 I would prefer that. the error case is that if someone parses HTML with things that look like HTML tags inside of a `STYLE` element or the other `RAWTEXT` elements in this patch, then the HTML API will think they are real tags.

Reply:
Judging from chatter in the `#6-3-release-leads` channel, it looks as if there's plans for a 6.3.2. I'll set that as the milestone for the ticket then and will backport the changeset to the 6.3 branch 👍

Reply:
thanks. I'll make sure to apply this one to the 6.3 compat folder in Gutenberg as well then